firefox_puppeteer/firefox_puppeteer/api/appinfo.py

```diff
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from firefox_puppeteer.base import BaseLib
+from ..base import BaseLib
```

Comment:
I digged around a bit for absolute vs. relative imports and found the Python docs: https://docs.python.org/2/whatsnew/2.5.html#pep-328-absolute-and-relative-imports

I will check all packages accordingly.